apps/mylar3/docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: "3"
+
+services:
+  mylar3:
+    container_name: mylar3
+    image: lscr.io/linuxserver/mylar3:v0.7.1-ls84
+    ports:
+      - ${APP_PORT}:8090
+    volumes:
+      - ${APP_DATA_DIR}/data/mylar3-config:/config
+      - ${ROOT_FOLDER_HOST}/media/data/comics:/comics
+      - ${ROOT_FOLDER_HOST}/media/downloads/mylar3:/downloads 
```

Comment:
Are we sure we want the subfolder `/mylar3`?

Reply:
It was to organize the `Downloads` Folder.

I added the `Downloads` folder for apps that are not under the Torrent or Usenet Section.

So It could be from any type of Download app, we can organize the folder.

Reply:
Cool, then it's all good for me